docs/book/tutorials/deploying_cpi_with_multi_dc_vc_aka_zones.md

```diff
@@ -1,14 +1,14 @@
-# Deploying the vSphere CPI and CSI in a Multi-vCenter OR Multi-Datacenter Environment using Zones
+# Deploying the vSphere CPI in a Multi-vCenter OR Multi-Datacenter Environment using Zones
```

Comment:
This one involves both CPI and CSI, CSI will be responsible for managing the persistent storage in k8s.
Instead of removing CSI in this documentation, I think it's better to add a link to CSI repo about how to install it